must-gather/collection-scripts/pre-install.sh

```diff
@@ -77,6 +77,21 @@ cleanup() {
   fi
 }
 
+# Delete all openshift-must-gather namespaces
+delete_mg_namespaces () {
+    namespaces=$(oc get namespaces -o name)
+    namespace_list=$(echo $namespaces | tr ";" "\n")
+    for namespace in ${namespace_list}; do
+        if [[ "$namespace" == *"openshift-must-gather"* ]]; then
```

Comment:
This is dangerous if customer will run OCP and OCS must gather in parallel, then you will delete one of the running one.